evpn-mh.clab.yml

```diff
@@ -0,0 +1,51 @@
+name: evpn-mh
+
+topology:
+  kinds:
+    srl:
+      image: ghcr.io/nokia/srlinux:23.3.1
+    linux:
+      image: akpinar/alpine:latest 
```

Comment:
for hackathon we copied your image under a common name `srl-labs/alpine:latest` let's use it here.
Later we will need to create dockerfile for it, so that we capture what is installed there and control the enhancements with tagging/versioning

Reply:
do you know its credentials for ssh?

Reply:
`srllabs@123`

Reply:
thx! verified the lab with the new images.